packages/flutter/lib/src/widgets/basic.dart

```diff
@@ -4497,66 +4497,452 @@ class PositionedDirectional extends StatelessWidget {
+///      much space is placed in front of and behind each child. This
+///      causes the empty spaces on each end to be
+///      half as large as the empty spaces between children, like this:
+///      Space-Child-Space    Space-Child-Space    Space-Child-Space
+///
+///      You can think of this like:
+///      _"[spaceAround] = The space around each child is the same"_.
+///
+///    Cross Axis:
+///    - [CrossAxisAlignment.start]: Similar to [MainAxisAlignment.start],
+///      the children will be aligned to the top for a [Row], and for a
+///      [Column] the child will be aligned to the left if the user's
+///      language reads ltr and to the right if the user's language reads
+///      rtl.
+///
+///    - [CrossAxisAlignment.center]: Similar to [MainAxisAlignment.center],
+///      the children will be centered in the cross axis. The largest child
+///      will touch the "sides" of a [Column] or "top and bottom" of a [Row],
+///      with all other children being their normal size in that direction
+///      _unless_ the cross axis maximum constraint passed in by the parent
+///      of the [Flex] is larger than the cross axis size of the largest child.
+///      In that case, no children will touch the cross axis sides of the
+///      [Flex] and there will be empty space around all children, and this
+///      space is _inside_ the [Flex].
+///
+///    - [CrossAxisAlignment.end]: Similar to [MainAxisAlignment.end],
+///      the children will be aligned to the bottom for a [Row], to the
+///      right if the user's language reads ltr and to the left if the
+///      user's language reads rtl.
+///
+///    - [CrossAxisAlignment.stretch]: All children will be given tight
+///      constraints in the cross axis, and those constraints will force
+///      all children to be as large as possible in the cross axis. The
+///      main axis sizes of the children will be the sizes calculated
+///      above in the main axis, potentially distorting the children by
+///      stretching them in only one direction.
+///
+///    - [CrossAxisAlignment.baseline]: Baseline is unique and tricky.
+///      For a [Row], the largest child is aligned with its top edge
+///      touching the top edge of the [Row]. Then, all children are
+///      aligned so their bottom edges are all aligned with the bottom
+///      edge _of the largest child_. If the [Row] is taller than the height
+///      of its tallest child, then no children will touch the bottom
+///      of the [Row] and it will appear that all children are "floating"
+///      above the bottom of the [Row]. For a [Column] the process is the
+///      same, however instead of the top and bottom edges, the [Column]
+///      aligns the largest child in the cross axis so it touches the "end"
+///      edge of the [Column] and then aligns "start side" all other children
+///      to the "start side" edge of the largest child.
+///
+/// Tips:
```

Comment:
```suggestion
/// ## Tips
///
```
Insert a blank comment line after this, and make it into a markdown heading without the colon.